docs/facts/known_issues.rst

```diff
@@ -7,34 +7,128 @@ This is the list of issues which are not or will not be fixed (yet...).
 * Under GitLab CI/CD the coloration may not be shown.
 
 
+Python < 3.7
+^^^^^^^^^^^^
+
+    .. versionchanged:: 4.0.0
+
+As of version 4 we no `longer support <../installation/index.html#python-3-7-0>`_
+python prior to version 3.7.
+
+This means you actually are unable to run with any version below python 3.7
+as a number of build-in features are missing and first introduced in 3.7
+
+The error message you might experience can be:
+
+.. code-block::
+
+    `Fatal Error: type object 'datetime.datetime' has no attribute 'fromisoformat'`
+
+.. code-block::
+
+    `ModuleNotFoundError: No module named 'dataclasses'`
+
+This can typically happens if you are using Ubuntu 18.x or 19.x
+
 Ubuntu 20.04.1 LTS Focal
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. versionadded:: 3.2.0
 
-In Ubunto release 20.04 they have removed a package name
+In Ubuntu release 20.04 they have removed a package name
 :code:`libffi.so.6` and upgraded it with version :code:`libffi.so.7`
 
 This means PyFunceble will trow an error like:
 
-:code:`ImportError: libffi.so.6: cannot open shared object file: No such file or directory`
+.. code-block:: console
+
+    ImportError: libffi.so.6: cannot open shared object file: No such file or directory
 
 The fix for this issue is then rather simple, add a softlink between the
 versions with :code:`ln -s`
 
 The complete line in my case was:
 
-:code:`sudo ln -s /usr/lib/x86_64-linux-gnu/libffi.so.7 /usr/lib/x86_64-linux-gnu/libffi.so.6`
+.. code-block:: console
+
+    sudo ln -s /usr/lib/x86_64-linux-gnu/libffi.so.7 /usr/lib/x86_64-linux-gnu/libffi.so.6
+
+However, the right way to do this is by first locate where your
+:code:`libffi.so.7` is with find
 
-However, the right way to do this is by first locate where your's
-:code:`libffi.so.7` is by
+.. code-block:: console
 
-:code:`find /usr/lib/ -type f -iname 'libffi.so.*'` and then apply the
-softlink to :code:`libffi.so.7`
+    find /usr/lib/ -type f -iname 'libffi.so.*'
+    
+Then apply the softlink to :code:`libffi.so.7`
 
 
 Combination of :code:`-f`, :code:`-uf` and :code:`--adblock`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can not combine the usage of :code:`-f`, :code:`-uf` and :code:`--adblock`
-at the same time
+You can not combine the usage of :code:`-f`, :code:`-uf` with :code:`--adblock`
+simultaneously.
+
+
+------
+
+.. sectionauthor:: @spirillen
+
+PyFunceble v 3.2.x
+^^^^^^^^^^^^^^^^^^
+
+.. versionchanged:: 3.2.x
+
+.. deprecated:: 4.0.0.a1
+
+When you are using the HTTP status code module you might experience the
+following error corses by :code:`urllib3`
+
+.. code-block::
+
+    HTTPSConnectionPool(host='pyfunceble-not-resolved', port=443
```

Comment:
Let me explain this so that we can get toward our clarity goal hehe 😄 

Because PyFunceble will create its own DNS resolver and we want everything to work accordingly, I had to trick the `requests` module.

Indeed, while using the `requests` module to get the HTTP status code or the mined dataset, our implementation of the requests module (search for the HTTP adapter if you want the source code) will do the DNS resolution with our very own resolver.
But, if the resolver failed (cf: no `A` or `AAAA`), we will use the `pyfunceble-not-resolved` "domain" as placeholder. That will make requests look for that domain, but it won't find it because it is something nobody knows. 

That's why the `pyfunceble-not-resolved` domain in any version before 4.0.0.  This "error" was actually none - in my opinion. Just requests or better said in this case urllib3 not liking the way I tricked it. 

In version prior to `4.0.0a1`, things are a bit different. Because I don't that "domain" do be predictable, on each session , PyFunceble will generate something like ([see here](https://github.com/funilrys/PyFunceble/blob/b18b220d283b4cfdcb92eb70cdb1f3d37d2f9b61/PyFunceble/storage.py#L90)):

```python
>>> import secrets
>>> f"pyfunceble-{secrets.token_hex(12)}.com"
'pyfunceble-fcf0ab7db58a4cfdfd17204f.com'
>>> f"pyfunceble-{secrets.token_hex(12)}.com"
'pyfunceble-ab2b9257292958e4041a5f4c.com'
```

And apparently, the exception also disappeared from the version prior to `4.0.0a1`.

That's the little "story" about that `pyfunceble-not-resolved` thing 😄

Reply:
Yeah, but something entirely diff., is I should move it to the 3.x docs.... It doesn't belongs in the 4.x docs....